packages/client/helpers/generator-test/index.ts

```diff
@@ -0,0 +1,44 @@
+'use strict'
```

Comment:
It should be emitted by TypeScript automatically since we don't have `"alwaysStrict": false"` in `tsconfig.json`